op-node/flags/flags.go

```diff
@@ -59,6 +59,11 @@ var (
 		Usage:   "Enable the admin API (experimental)",
 		EnvVars: prefixEnvVars("RPC_ENABLE_ADMIN"),
 	}
+	RPCAdminPersistence = &cli.StringFlag{
+		Name:    "rpc.admin-state",
+		Usage:   "File path used to persist state changes made via the admin API so they persist across restarts",
+		EnvVars: prefixEnvVars("RPC_ADMIN_STATE"),
+	}
 
 	/* Optional Flags */
```

Comment:
It's an optional flag and in the optional-flags list, but was grouped with the required-flag definitions here.

Reply:
ah, I just put it with `rpc.enable-admin` which is also optional.  I'll move the optional flags comment above both.